ydb/_topic_writer/topic_writer_asyncio.py

```diff
@@ -252,6 +252,8 @@ async def close(self, flush: bool):
             self._check_stop()
         except TopicWriterStopped:
             pass
+        finally:
```

Comment:
Why we need flush second time here?

Reply:
When we close with flush=False and we canceled tasks do we need to wait futures?

Reply:
No, we don't.

All result features will set exception in _stop call (few lines above).